build.gradle

```diff
@@ -16,10 +16,9 @@ allprojects {
     repositories {
         google()
         mavenCentral()
-        jcenter() // Warning: this repository is going to shut down soon
```

Comment:
is this change fine?

Reply:
yes, we've cheeked with lucho, and jcenter() will be deprecated, mavenCentral() now take care of that function, the IDE recommended to delete that line, works fine.